torchao/experimental/kernels/cpu/aarch64/tests/test_utils.h

```diff
@@ -249,6 +277,44 @@ struct channelwise_8bit_activation_groupwise_lowbit_weight_test_case {
       }
     }
 
+#if 0 // Alternate reference implementation for debugging.
```

Comment:
remove

Reply:
I know though useful it will be a headache if we don't test it. Will remove.